tests/integration/utils.py

```diff
@@ -158,6 +193,10 @@ def __check_attributes_schema(actual_attributes, expected_attributes):
             expected_type = __get_attribute_type(expected_attribute)
             self.assertIsInstance(actual_value, expected_type)
 
+            # If null values are allowed, check schema with:
+            # if actual_value and expected_type:
+            #     self.assertIsInstance(actual_value, expected_type)
+
```

Comment:
Instead of commenting this out, you can control to logic by using a boolean value maybe called `null_value_allowed` and put it into the condition check as well.